src/store/actions.ts

```diff
@@ -485,6 +485,11 @@ export const resubmit: any = async ({ commit, getters }): Promise<boolean> => {
   // override request action code
   nrTypeData['request_action_cd'] = RequestCode.RESUBMIT
 
+  // override applicants partyIds
+  for (let [key, value] of Object.entries(nrTypeData['applicants'])) {
+    value['partyId'] = ''
```

Comment:
What are the id's used for?

Reply:
They come as part of the request with the ids for the old NR. In a submission we are creating a new NR and it needs a new applicant record in the DB. With the ids populated it doesn't create new applicants.